screenpipe-server/src/bin/screenpipe-server.rs

```diff
@@ -114,9 +114,12 @@ fn setup_logging(local_data_dir: &PathBuf, cli: &Cli) -> anyhow::Result<WorkerGu
         env_filter
     };
 
+    let timer =
+        tracing_subscriber::fmt::time::ChronoLocal::new("%Y-%m-%dT%H:%M:%S%.6fZ".to_string());
+
     let registry = tracing_subscriber::registry()
         .with(env_filter)
-        .with(fmt::layer().with_writer(std::io::stdout))
+        .with(fmt::layer().with_writer(std::io::stdout).with_timer(timer))
```

Comment:
any issue with the log component in app?

Reply:
nice catch! the `with_timer` needs to be applied to each layer.

working fine now
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/40e101e6-aed2-4c06-97fd-63f17f6e5a82" />